content/docs/studio/troubleshooting.md

```diff
@@ -237,6 +238,16 @@ its parent Git commit is present in the project table. So, before you run the
 experiment, make sure that its parent commit is pushed to Git and shown in the
 project table.
 
+## Project does not display DVC experiments
+
+Iterative Studio automatically checks for updates to your repository using
+webhooks, but it can not rely on this mechanism for custom git objects, like DVC
+experiment references. So the experiments you push using `dvc exp push` may not
+automatically display in your project table.
```

Comment:
```suggestion
automatically display in your project table unless you [configure the Studio token](/doc/user-guide/project-structure/configuration#studio).
```

Reply:
I think it's ok not to write about the token here. The experiments may not automatically appear in Studio for other reasons as well (eg, if there's a connection issue between the user's machine and Studio).